mapper/pchain/tx_parser_test.go

```diff
@@ -0,0 +1,293 @@
+package pchain
+
+import (
+	"testing"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/avalanchego/vms/components/avax"
+	"github.com/coinbase/rosetta-sdk-go/types"
+	"github.com/stretchr/testify/assert"
```

Comment:
nit: we should probably use `"github.com/stretchr/testify/require"` instead -- it's semantically the same as `assert` but fails immediately on an error

Reply:
Considering the existing tests also use `assert`, how about a separate PR to migrate all tests to use require instead?